.checkstyle/checkstyle.xml

```diff
@@ -107,7 +107,7 @@
 
         <module name="ClassFanOutComplexity">
             <!-- default is 20 -->
-            <property name="max" value="44"/>
+            <property name="max" value="45"/>
```

Comment:
Would using an annotation to suppress the issue make more sense than changing the defaults?

Reply:
Agree. I reverted it back, but the suppress annotation seems to be not needed anymore. I guess at some point I changed the code and increasing the fan out complexity was not needed anymore :-)